.gitignore

```diff
@@ -113,3 +113,6 @@ WordPress/InfoPlist-internal.h
 
 # SwiftLint Remote Config Cache
 .swiftlint/RemoteConfigCache
+
+# Gutenberg extra bundle and source map files
+gutenberg/react-native-bundle-source-map
```

Comment:
I wonder if this is still needed after the path was changed to the Pods folder? Maybe we can remove it?

Reply:
Oh, good catch. Yes, we no longer need it so I'll remove it.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22862/commits/9934e245c4bf82b93c591179676b4e8eb09afb35.